tests/CI/ci_test.sh

```diff
@@ -1,34 +1,44 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
 PILOTBRANCH=pilot_loggers
-DEBUG=True
 
 echo -e "**** Starting Pilot tests with PilotLogger ****\n"
 
-echo -e '***' $(date -u) "**** Getting the tests ****\n"
+echo -e "***' $(date -u) **** Getting the tests ****\n"
 
-mkdir -p $PWD/TestCode
-cd $PWD/TestCode
+mkdir -p "$PWD/TestCode"
+if ! cd "$PWD/TestCode"; then
+  exit 1
+fi
 
 git clone https://github.com/wkrzemien/Pilot.git
-cd Pilot
-git checkout $PILOTBRANCH
-echo `pwd`
-cd ../..
-echo `pwd`
-
-echo -e '***' $(date -u) "**** Got the tests ****\n"
-
-source TestCode/Pilot/tests/CI/pilot_ci.sh 
+if ! cd Pilot; then
+  exit 1
```

Comment:
Instead of testing each line you could use `set -e` for all that preparation section